docs/superpowers/plans/2026-05-29-opencode-compatibility.md

```
@@ -0,0 +1,579 @@
+- Modify: `README.ko-KR.md`
+- Modify: `README.ru.md`
+- Modify: `README.es.md`
+- Modify: `README.cn.md`
+
+- [ ] **Step 1: Update each localized README's batch processing row**
+
+For each localized README, find the batch processing feature row that mentions `claude -p` and update it to be platform-agnostic.
+
+| File | Current | New |
+|------|---------|-----|
+| `README.zh-TW.md` | `使用 `claude -p` 工作器並行評估` | `使用 headless CLI 工作器並行評估（`claude -p`、`opencode run` 等）` |
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)